apps/template/src/App.tsx

```diff
@@ -39,20 +39,17 @@ export const App = (props: AppProps) => {
   })
 
   const fallbackRender = ({ error }: { error: { message: string } }) => {
-    return (
-      <div className="w-1/2">
-        <CodeBlock className={preErrorClasses} copy={false}>
-          {error?.message || error?.toString() || "An error occurred"}
-        </CodeBlock>
-      </div>
-    )
+    return <div className="w-1/2">{error?.message || error?.toString() || "An error occurred"}</div>
   }
 
   return (
     <QueryClientProvider client={queryClient}>
       <AppShell
         pageHeader="Converged Cloud | App Template"
         embedded={props.embedded === "true" || props.embedded === true}
+        fullWidthContent={false}
+        sideNavigation={null}
+        topNavigation={null}
```

Comment:
aren't these props optional why do we need to supply falsy values?

Reply:
For same weird reasons I will have a look how I can configure it better